src/masonite/validation/Validator.py

```diff
@@ -1029,6 +1029,46 @@ def negated_message(self, attribute):
         return "The {} is a valid {} postal code.".format(attribute, self.locale)
 
 
+def parse_table(table_name, column="id"):
+    import importlib
+    if "." in table_name:
+        m = importlib.import_module(table_name)
+        # get model
+        model_name = table_name.split(".")[-1]
+        model = getattr(m, model_name)
+        table = model().get_table()
+        return table, model
+    else:
+        return table_name, None
+
+
+class exists_in_db(BaseValidation):
+    def __init__(self, validations, table, column=None, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        # retrieving db connection will totally change with new ORM
+        # in addition, below is not really satisfying
+        from masonite.helpers import config
+        self.connection = config("database.db")
+        self.column = column
```

Comment:
I actually like how you did this so I guess we can keep doing down this right

Reply:
okay cool ! 
I will upgrade this to use masonite ORM and see if I can work it out for the different use cases.

maybe I should take care to ensure that validation can works even if ORM is not there or installed

Reply:
Yeah that's an important step. This package should be useable outside of even needing Masonite .. Having some methods where Masonite is required is the best of both world but shouldn't need a package to use this one